types/react-avatar-editor/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for react-avatar-editor 10.3
+// Type definitions for react-avatar-editor 11.0
```

Comment:
When you update a major version, please follow the guidance here: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version